src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -23,13 +23,32 @@
     private static final double WINDOW_WIDTH = 1000;
     private static final double WINDOW_HEIGHT = 500;
 
+    // login page vars
     private Stage thestage;                         // Holds the scene that user is currently viewing
     private Scene signInScene, mainScene;           // Scenes to hold all the elements for each page
     private Button signInButton;
     private TextField userInput;
     private PasswordField passInput;                // Takes input for username and password
     private ClientContext clientContext;            //
 
+    // main page vars
+    private HBox hboxClient;
+    private HBox hboxInput;
+    private VBox userVBox;
+    private VBox chatVBox;
+    private VBox convosVBox;
+    private BorderPane container;
+    private Button sendButton;
```

Comment:
I can try to help you split it up if you want, or you can continue working from one class.

To split it up, I'd create a class that basically handles everything in one `Scene`. Something like this:

```java
public class ChatScene{
  private Scene chatScene;

  public ChatScene(){
    //build chatScene
   }

  public Scene getScene(){
     return chatScene;
   }
}
```

Then when you want to use that, you would just create an instance and get the `Scene` from it in the main window. Something like this:

```java
public class MainWindow{
   private LoginScene loginScene;
   private ChatScene chatScene;

  public MainWindow(){
    //create main window, set the scene to `loginScene.getScene()`
   }

   public void loginSucess(){
      //switch scene to `chatScreen.getScene()`
   }
}
```

You might need some more connecting code (potentially passing the `MainWindow` into each scene so it can access stuff like the `loginSuccessful()` function), but that's the basics of how I'd split up GUI code. You might even go one level deeper and split it up into individual sections, but that might be overkill for this project.

Completely up to you guys how to proceed though.